sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md

```diff
@@ -9,6 +9,7 @@ csharp: true
 library-name: Subscription
 namespace: Azure.ResourceManager.Subscription
 require: https://github.com/Azure/azure-rest-api-specs/blob/2614c5038cb384e864c1608e76be61a5ee84cb02/specification/subscription/resource-manager/readme.md
+#tag: package-2021-10
```

Comment:
should be removed

Reply:
This helps to know the version without open the readme.md file

Reply:
Don't feel like it is a good thing to keep, because it is easy to get out-of-date.
We can leave it for discussion, will let it pass for this PR.